src/bridges/curve/CurveStEthLpBridge.vy

```diff
@@ -0,0 +1,202 @@
+            used to save gas.
+    """
+    ROLLUP_PROCESSOR = _rollupProcessor
+    LP_TOKEN = ICurvePool(CURVE_POOL).lp_token()
+
+    val: uint256 = max_value(uint256)
+    ERC20(STETH).approve(CURVE_POOL, val, default_return_value = True)
+    ERC20(STETH).approve(WSTETH, val, default_return_value = True)
+    ERC20(LP_TOKEN).approve(ROLLUP_PROCESSOR, val, default_return_value = True)
+    ERC20(WSTETH).approve(ROLLUP_PROCESSOR, val, default_return_value = True)
+
+    ISubsidy(SUBSIDY).setGasUsageAndMinGasPerMinute(0, 250000, 180)
+    ISubsidy(SUBSIDY).setGasUsageAndMinGasPerMinute(1, 250000, 180)
+
+@payable
+@external
+def __default__():
+    """
+    @notice Default used to accept Eth from pool when exiting
+    """
+    pass
+
+@payable
+@external
+def convert_11192637865(
+    _inputAssetA: AztecAsset,
+    _inputAssetB: AztecAsset,
+    _outputAssetA: AztecAsset,
+    _outputAssetB: AztecAsset,
+    _totalInputValue: uint256,
+    _interactionNonce: uint256,
+    _auxData: uint64,
+    _rollupBeneficiary: address
+) -> (uint256, uint256, bool):
+    """
+    @notice Function called by the defi proxy, executes deposit or withdrawal depending on input 
+    @dev    Instead of `convert` named `convert_11192637865` to work around the `convert` keyword
+            while still having the same selector.
+    @param  _inputAssetA - The first Aztec Asset to input the call, will be LPToken or ETH or WSTETH
+    @param  _inputAssetB - Always empty for this bridge
+    @param  _outputAssetA - The first aztec asset to receive from the call, will be LPToken or ETH
+    @param  _outputAssetB - The second aztec asset to receive from the call, will be none or WSTETH
+    @param  _totalInputValue - The amount of token to deposit or withdraw
+    @param  _interactionNonce - The unique identifier for this defi interaction
+    @param  _auxData - Auxiliary data that can be used by the bridge
+    @param  _rollupBeneficiary - The address of the beneficiary of subsidies
+    @return outputValueA - The amount of outputAssetA that the rollup should pull
+    @return outputValueB - The amount of outputAssetB that the rollup should pull
+    @return isAsync - True if the bridge is async, false otherwise. Always false for this bridge
+    """
+    assert msg.sender == ROLLUP_PROCESSOR, "Invalid caller"
+
+    if _inputAssetB.assetType != 0:
+        raise "Invalid asset B"
+
+    # Eth or WSTETH in -> LPToken out
+    deposit: bool = (_inputAssetA.assetType == 1 or (_inputAssetA.assetType == 2 and _inputAssetA.erc20Address == WSTETH)) and _outputAssetA.assetType == 2 and _outputAssetA.erc20Address == LP_TOKEN
+
+    # LPToken in -> ETH + WSTETH out
+    withdraw: bool = _inputAssetA.assetType == 2 and _inputAssetA.erc20Address == LP_TOKEN and _outputAssetA.assetType == 1 and _outputAssetB.assetType == 2 and _outputAssetB.erc20Address == WSTETH
+
+    if not((deposit or withdraw) and not(deposit and withdraw)):
```

Comment:
can you instead use something simpler like:

`assert deposit == withdraw`

Reply:
Simplified with `assert deposit != withdraw, "Invalid assets"` in https://github.com/AztecProtocol/aztec-connect-bridges/pull/229/commits/025335a46cb49467877c80995aadab3af064aca4